src/globals/scss/_typography.scss

```diff
@@ -32,8 +32,12 @@ $major-second-typescale-map: (
   @return ($px / $base-font-size) * 1em;
 }
 
-@mixin helvetica {
-  font-family: 'IBM Helvetica', Helvetica Neue, HelveticaNeue, Helvetica, sans-serif;
+@mixin font-family {
+  @if global-variable-exists('css--plex') and $css--plex == true {
+    font-family: 'IBM Plex Sans', sans-serif;
```

Comment:
Should we use the fallback stack that James has in IBM/type, or is this a preference for Carbon-related stuff?

Reference: https://github.com/IBM/type/blob/master/src/_core.scss#L8